README.md

```diff
@@ -1,7 +1,6 @@
-<a href="http://www.tombolo.org.uk/products/">
-<p align="center"> <img src="http://www.tombolo.org.uk/wp-content/uploads/2017/09/xDigital-Connector-Icon.png.pagespeed.ic.17pEUiGbW4.png" width="256" height="256"/>
-</p></a>
-<h1 align="center"> Tombolo Digital Connector</h1>
+[![Tombolo](http://www.tombolo.org.uk/wp-content/uploads/2017/09/xDigital-Connector-Icon.png.pagespeed.ic.17pEUiGbW4.png)](http://www.tombolo.org.uk/products/)
```

Comment:
I uploaded a 256x256 icon on the issue #369. You could use that here so it's the size of the one before.